tests/e2e-kubernetes/testsuites/util.go

```diff
@@ -105,6 +106,14 @@ func createVolumeResource(ctx context.Context, config *storageframework.PerTestC
 	pvName := "s3-e2e-pv-" + uuid.New().String()
 	pvcName := "s3-e2e-pvc-" + uuid.New().String()
 
+	// Add debug options if they're not already present
+	debugOptions := []string{"debug", "debug-crt"}
+	for _, option := range debugOptions {
+		if !slices.Contains(mountOptions, option) {
```

Comment:
Maybe we should parse `mountOptions` with [`mountpoint.ParseArgs`](https://github.com/awslabs/mountpoint-s3-csi-driver/blob/main/pkg/mountpoint/args.go#L54) here and use that to query presence of an option? If the caller uses `--debug` to enable debug logs, this would duplicate it for example.